charts/navi-castle/values.yaml

```
@@ -121,38 +121,54 @@ resources: {}
 # @section Navi-Castle service settings
 
 # @param castle.castleDataPath Path to the data directory.
+# @param castle.excludeProjects Array of project labels to exclude
 # @extra castle.restrictions Section ignored if castle.restriction.enabled=false
 # @param castle.restrictions.host Restrictions API base URL.
 # @param castle.restrictions.key Restrictions API key.
 # @param castle.jobs Number of parallel downloading jobs.
+# @param castle.logLevel Logging level, one of: DEBUG, INFO, WARNING, ERROR, CRITICAL.
 # @extra castle.startupProbe [nullable] Settings for startup probes
 # @param castle.startupProbe.periodSeconds Check period for startup probes.
 # @param castle.startupProbe.failureThreshold Threshold for startup probes.
+# @extra castle.storePeriod [nullable] Retention period for the corresponding data if enabled, ref. `cron`, `init` and `rtr` sections. Supported values: `day`, `month`, `week` or a specific number of days.
+# @param castle.storePeriod.import Retention period for `import` job data, ref. `cron/init.enabled.import`
+# @param castle.storePeriod.restriction Retention period for `restriction` job data, ref. `cron/init.enabled.restriction`
+# @param castle.storePeriod.restrictionImport Retention period for `restrictionImport` job data, ref. `cron/init.enabled.restrictionImport`
+# @param castle.storePeriod.rtr Retention period for `rtr` data, ref. `rtr.enabled`
 
 castle:
   image:
     repository: 2gis-on-premise/navi-castle
     pullPolicy: IfNotPresent
-    tag: 1.9.2
+    tag: 1.9.8
   castleDataPath: /opt/castle/data/
   restrictions:
     host: http://restrictions-api.host
```

можем это сразу поправить и добавить в Breaking Changes?
```suggestion
    url: ''
```

---

убрал

---

а host -> url?

---

не заметил, переименовал

---

в values все так же host)

---

закоммитил третью попытку